src/components/EResourceSections/PackageContents.js

```diff
@@ -24,11 +25,23 @@ export default class PackageContents extends React.Component {
       packageContentsFilter: PropTypes.string,
     }),
     id: PropTypes.string,
+    isPending: PropTypes.bool,
     onFilterPackageContents: PropTypes.func.isRequired,
+    onNeedMorePackageContents: PropTypes.func.isRequired,
     onToggle: PropTypes.func,
     open: PropTypes.bool,
   };
 
+  state = {};
+
+  static getDerivedStateFromProps(props, state) {
```

Comment:
Yeah I think I just became more in favour of https://github.com/folio-org/stripes-connect/pull/124 if it lets us avoid logic like this 😆